softwareComponents/rofiHalInc/include/span.hpp

```diff
@@ -0,0 +1,618 @@
+// as soon as rofiHalEsp32 supports C++20 this dependency may be removed in favor of standard implementation of std::span
```

Comment:
Tento soubor patří spíš jinam. Můžeš ho asi dát do atoms

Reply:
removed in favor of c++20